docs/use-the-network/run-a-network-server/debug-with-sniffer.mdx

```diff
@@ -0,0 +1,280 @@
+---
+id: debug-with-sniffer
+sidebar_label: Debug With Sniffer
+slug: /use-the-network/debug-with-sniffer
+---
+
+# Debug With Sniffer
+
+Confirm router runtime behavior using a virtual LoRaWAN device with a
+software-based protocol sniffer.
+
+## Dependencies
+
+1. Ensure [Router](https://github.com/helium/router.git) is running and
+   [Console](https://github.com/helium/console.git) is connected to it.
+1. The wallet to be used requires enough HNT to burn for joining the virtual
+   device to the network.
+   + It's the same fee as for a physical device.
+   + See
+   [Setup Data Only Hotspot](https://docs.helium.com/mine-hnt/data-only-hotspots/#transactions--cost)
+1. Clone each of these repos:
+    + [virtual-lorawan-device](https://github.com/helium/virtual-lorawan-device)
+    + [lorawan-sniffer](https://github.com/helium/lorawan-sniffer)
+    + [gateway-rs](https://github.com/helium/gateway-rs)
+    + [helium-wallet-rs](https://github.com/helium/helium-wallet-rs)
+
+Those packages require [Rust](https://rust-lang.org/) when building from
+source.
+
+Everything may run locally on a laptop/workstation, or use just the
+Rust-based tools locally for connecting to a remote Router & Console.
+
+## Overview
+
+The approximate sequence:
+
+1. Create a virtual device
```

Comment:
all 1s?

Reply:
Fixed in commit be584ee.

FYI:

Markdown doesn't care about the actual number used, and each list gets rendered correctly:
https://deploy-preview-794--helium-docs.netlify.app/use-the-network/debug-with-sniffer/#overview

Probably only doc maintainers will ever see inside the .mdx file, so this is a technique for keeping future diffs cleaner, making future PRs easier/faster to review.